### DISCUSSION TITLE
How to execute multiple tasks concurrently

### DISCUSSION THREAD

**Discussion by YishuiLi** [PrefectHQ/prefect#3851] [General]
Here is my code, there are four tasks (start, get_first_name, get_last_name, print_info). I want to execute get_first_name and get_last_name concurrently. How to implement it with prefect？
````
from prefect import task, Flow
from prefect.run_configs import LocalRun

@task(log_stdout=True)
def start():
    return "Jackie Lee"

@task(log_stdout=True)
def get_first_name(name):
    return name.split(" ")[0]

@task(log_stdout=True)
def get_last_name(name):
    return name.split(" ")[1]

@task(log_stdout=True)
def print_info(fname, lname):
    print("first name is " + fname + ", last name is " + lname)

with Flow("test_parallel") as flow:
    name = start()
    fname=get_first_name(name)
    lname=get_last_name(name)
    print_info(fname, lname)

flow.run_config = LocalRun()
flow.register(project_name="test")
````
![image](https://user-images.githubusercontent.com/16732930/102299354-c8660580-3f8d-11eb-885a-822002feac49.png)

**Comment by cicdw** [comment]
Hi @YishuiLi - to specify how you want your tasks executed, you can use an `executor`.  In your case, the `LocalDaskExecutor` should do the trick:
```python
from prefect.engine.executors import LocalDaskExecutor

flow.run_config = LocalRun()
flow.executor = LocalDaskExecutor()
flow.register(project_name="test")
```

For more info, check out these docs: https://docs.prefect.io/orchestration/flow_config/executors.html

**Reply by YishuiLi** [reply]
It worked! Thank you!
